src/main/java/stream/flarebot/flarebot/commands/general/ConvertCommand.java

```diff
@@ -0,0 +1,108 @@
+package stream.flarebot.flarebot.commands.general;
+
+import net.dv8tion.jda.core.EmbedBuilder;
+import net.dv8tion.jda.core.entities.Member;
+import net.dv8tion.jda.core.entities.Message;
+import net.dv8tion.jda.core.entities.MessageEmbed;
+import net.dv8tion.jda.core.entities.TextChannel;
+import net.dv8tion.jda.core.entities.User;
+import stream.flarebot.flarebot.FlareBot;
+import stream.flarebot.flarebot.commands.Command;
+import stream.flarebot.flarebot.commands.CommandType;
+import stream.flarebot.flarebot.objects.GuildWrapper;
+import stream.flarebot.flarebot.util.MessageUtils;
+import stream.flarebot.flarebot.util.currency.CurrencyComparison;
+import stream.flarebot.flarebot.util.currency.CurrencyConversionUtil;
+
+import java.awt.Color;
+import java.io.IOException;
+import java.text.DecimalFormat;
+import java.util.Random;
+
+public class ConvertCommand implements Command {
+
+    private Random random = new Random();
+    public static final DecimalFormat DECIMAL_FORMAT = new DecimalFormat("0.#################");
```

Comment:
Preferable to keep these separate, different scopes and a static final variable.

Reply:
Not sure what you mean...?

Reply:
Just add a space between them

Reply:
Oh ok xD